pkg/services/ecs/cloudconfig.go

```diff
@@ -50,7 +50,9 @@ func (c *CloudConfig) genCloudProviderSecretTask() (writeFile *WriteFile, runCmd
 	}
 
 	runCmd = []string{
-		"if ! kubectl get secret cloud-config; then kubectl create secret generic cloud-config --from-file=/etc/kubernetes/cloud-config; fi",
+		"sleep 10",
```

Comment:
Does this mean that we should wait for something to be ready? 
- better add a comment about it or add a check for that here

Reply:
TODO is added now, and this should be optimized later 🤝